code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -967,3 +967,74 @@
 				to_chat(M, "<span class='warning'>Your missing arm aches from wherever you left it.</span>")
 				M.emote("sigh")
 	return ..()
+
+/datum/reagent/toxin/potash
+	name = "Potash"
+	id = "potash"
+	description = "An odorless white crystal resembling and tasting of salt. Used to cause heart arrhythmias in high doses or cardiac arrest through injection. Also an effective fertiliser"
+	taste_description = "salt"
+	reagent_state = SOLID
+	color = "#FFFFFF" // rgb: 255,255,255
+	toxpwr = 0
+	metabolization_rate = 0.4
+	var/amount_to_convert = 0
+
+/datum/reagent/toxin/potash/reaction_mob(mob/living/carbon/M, method=TOUCH, reac_volume)
+	if (method==INJECT)
+		M.reagents.add_reagent("injectedpotash", reac_volume)
```

Comment:
Not sure if intended, but I think this'll lead to having both potash and injectedpotash in the system after injection.